gateway/src/test/kotlin/com/example/gateway/grpc/GrpcPassServiceTest.kt

```diff
@@ -1,41 +1,33 @@
 package com.example.gateway.grpc
```

Comment:
the class `GrpcPassService` has been moved to another package, but the test hasn't